qa/os/src/test/java/org/elasticsearch/packaging/util/ProcessInfo.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.packaging.util;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+
+/**
+ * Encapsulates the fetching of information about a running process.
```

Comment:
I believe this javadoc should mention that the code here is targeting Linux, and perhaps mention the specific distro or environment it's intended for. The commands here wouldn't work on a Mac (no `/proc` directory, for one thing), and certainly wouldn't work on Windows, and someone new to the code might think this is a general class for retrieving process info.

Reply:
Good point, I'll update the Javadoc.